lib/runtime-c-api/wasmer.hh

```diff
@@ -612,32 +635,46 @@ uint32_t wasmer_table_length(wasmer_table_t *table);
 /// and `wasmer_last_error_message` to get an error message.
 wasmer_result_t wasmer_table_new(wasmer_table_t **table, wasmer_limits_t limits);
 
+#if (!defined(_WIN32) && defined(ARCH_X86_64))
```

Comment:
Optional: these are clearly a group, just put a single #if around them to disable all these related trampoline functions for windows?

Reply:
Yeah, it's unfortunate that it did that; if we manually edit the header file though then we have to do it after every time it's generated

Reply:
I put it at the top level in Rust